cli/internal/lockfile/berry_lockfile.go

```diff
@@ -104,3 +116,17 @@ func DecodeBerryLockfile(contents []byte) (*BerryLockfile, error) {
 	prettyLockFile := BerryLockfile(yarnSplitOutEntries(lockfile))
 	return &prettyLockFile, nil
 }
+
+func yarnSplitOutEntries(lockfile map[string]*YarnLockfileEntry) map[string]*YarnLockfileEntry {
+	prettyLockfile := map[string]*YarnLockfileEntry{}
+	// This final step is important, it splits any deps with multiple-resolutions
+	// (e.g. "@babel/generator@^7.13.0, @babel/generator@^7.13.9":) into separate
+	// entries in our map
+	// TODO: make concurrent
```

Comment:
There is just no way I can imagine that concurrency on this tiny task is going to be a win.
```suggestion
```